pages/components/WizardExample.js

```diff
@@ -23,7 +23,7 @@ const WizardExample = () => {
   }
   return (
     <>
-      <Wizard onComplete={onComplete} debug>
+      <Wizard onComplete={onComplete} updateUrl debug>
```

Comment:
I think you forgot to update here

```suggestion
      <Wizard onComplete={onComplete} stateManager="url" debug>
```

Reply:
Thanks!